src/assets/_project/_blocks/scss-general/_qg-variables.scss

```diff
@@ -355,5 +357,6 @@ $text-underline-offset: 5px;
   // unless we're inside a context where exact positioning matters
   &:not(#ict-tabcontainer *, #tabbed-nav *) {
     margin: $margin;
+    margin-left: 0
```

Comment:
This will affect just about every link and button on the site. What is the purpose of it?